third-party/eazy/v1/openapi.yaml

```diff
@@ -2572,7 +2572,7 @@ components:
           description: Array of event on the webhooks
           type: array
           items:
-            $ref: '#/components/schemas/events'
+            $ref: '#/components/schemas/event'
```

Comment:
Is this correct?

Reply:
just see that this is major mistake

it's a list of strings

<html>
<body>
<!--StartFragment-->

custom | Custom event for specific customer purposes
-- | --
echo | Outgoing messages sent by agents or assistants
handover | Handover event whenever a conversation is transferred to an assistant or agent
label | Label event whenever a conversation is labelled or unlabelled
message | Incoming messages from customers
note | Note creation or deletion event
postback | Postback event for WhatsApp button templates, Apple list pickers etc
receipt | Message receipt event
template_pack | Template updates whenever templates are approved, rejected etc.
typing | Typing indications when customers are typing a message

<!--EndFragment-->
</body>
</html>